alembic/versions/c819d65fd831_add_news_flash_qualification.py

```diff
@@ -15,10 +15,9 @@
 from alembic import op
 import sqlalchemy as sa
 
-from anyway.backend_constants import BE_CONST, NewsflashLocationQualification
-
 
 def upgrade():
+    from anyway.backend_constants import NewsflashLocationQualification
```

Comment:
What is the reason for moving import from beginning of file?

Reply:
when i tried to add the new table, the alembic was not able to import the package anyway.
i also so similar behavior at this file:
https://github.com/hasadna/anyway/blob/0c0c4018b44a7da79c9654c9d9654d4f186e5ef1/alembic/versions/bd67c88713b8_user_permissions_management.py#L69